testcafe/tests/pages/viewer/viewer-model.js

```diff
@@ -73,6 +74,33 @@ export default class Viewer {
         .expect(this.viewerNavPreviousBtn.visible)
         .ok('Previous arrow does not show up')
         .click(this.viewerNavPrevious)
+      await this.waitForLoading()
+    }
+  }
+
+  //@param {String} screenshotPath : path for screenshots taken in this test
+  //@param {number} startIndex : index of the 1st file to open
+  //@param {number} numberOfNavigation : the number of file we want to go through during the test.
+  async navigateInViewer(screenshotPath, startIndex, numberOfNavigation) {
+    console.log(
+      `startIndex : ${startIndex} / numberOfNavigation : ${numberOfNavigation}`
+    )
+    for (let i = startIndex; i < startIndex + numberOfNavigation; i++) {
+      await this.navigateToNextFile(i)
+      if (t.fixtureCtx.isVR)
+        await t.fixtureCtx.vr.takeScreenshotAndUpload(
+          `${screenshotPath}-${i}-next`
+        )
+    }
+
+    for (let i = startIndex + numberOfNavigation - 1; i > startIndex; i--) {
```

Comment:
The `navigateInViewer` method was moved from `drive-viewer/drive-viewer-model.js` to `viewer/viewer-model.js`in order to be used both in drive and photos, but that made me realize I add forget `-1` in my 2nd loop. (which trigger an error when starting navigation from 0 index)
as a consequence screenshots in the `public drive viewer` scenario will change, so I'll validate those screenshots as reference when merging this PR, to avoid false positive in between